scripts/source/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "fileprocessor",
```

Comment:
probably give it a different name

Reply:
Good call. And it really does make sense to unify them. I'll look at doing that next week when we tackle the other docs.